subjects/invert_sentence/README.md

```diff
@@ -0,0 +1,36 @@
+## invert_sentence
+
+### Instructions
+
+Write a function called `invert_sentence` that takes a string as input and returns the words in the string in reverse order.
+In other words, the function should take a sentence as input and return a new sentence with the words reversed.
+
+```rust
+pub fn invert_string(string: &str) -> String {
+    // Your code goes here
+}
+```
+
+### Usage
+
+Here is a possible runner to test your function :
+
+```rust
+use invert_sentence::invert_sentence;
+
+fn main() {
+    println!("{}", invert_sentence("Rust is Awesome"));
+    println!("{}", invert_sentence("   word1     word2  "));
```

Comment:
In my original idea I was thinking about preserving the number of white spaces in between words, which should make the exercise a bit harder to solve, otherwise we should classify it as easy rather than medium.

As an example word1   word2  =>   word2   word1